examples/acquiring_data/querying_the_GOES_event_list.py

```diff
@@ -22,34 +23,34 @@
 # We then use the client to query the HEK for a list of events
 # that were detected by the GOES X-ray Sensor (XRS) instrument between
 # `tstart` and `tend`.
-result = client.search(hek.attrs.Time(tstart, tend),
```

Comment:
would it not be better to transition the docs to using Fido?

Reply:
Fixed.